### DISCUSSION TITLE
Guest WireGuard mode

### DISCUSSION THREAD

**Discussion by ghost** [wg-easy/wg-easy#1059] [Ideas]
Any idea how to set internet only WG but block all lan access including port 22 ssh?
Tried with Iptables and ebtables but consider docker network ip is not stable after reboot.
It's just not practical to do so. I'm stumbled.

**Comment by eandersons** [comment]
You could use allowed IPs calculator by Pro Custodibus: https://www.procustodibus.com/blog/2021/03/wireguard-allowedips-calculator/

I strongly suggest to also read the section [A better alternative](https://www.procustodibus.com/blog/2021/03/wireguard-allowedips-calculator/#a-better-alternative) that could provide an additional solution.
